plugins/ruby-foreman-salt/debian/control

```diff
@@ -2,12 +2,12 @@ Source: ruby-foreman-salt
 Section: ruby
 Priority: extra
 Maintainer: Stephen Benjamin <stephen@redhat.com>
-Build-Depends: debhelper (>= 8.0.0), foreman-assets (>= 1.24.0), foreman (>= 1.24.0), foreman-nulldb (>= 1.24.0), ruby-foreman-deface, ruby-foreman-tasks (>= 0.14.5), ruby-foreman-remote-execution (>= 2.0.0)
+Build-Depends: debhelper (>= 8.0.0), foreman-assets (>= 3.1.0~rc1), foreman (>= 3.1.0~rc1), ruby-foreman-deface, ruby-foreman-tasks (>= 5.1.0), ruby-foreman-remote-execution (>= 4.4.0-1)
```

Comment:
```suggestion
Build-Depends: debhelper (>= 8.0.0), foreman-assets (>= 3.1.0~rc1), foreman (>= 3.1.0~rc1), foreman-nulldb (>= 3.1.0~rc1), ruby-foreman-deface, ruby-foreman-tasks (>= 5.1.0), ruby-foreman-remote-execution (>= 4.4.0-1)
```

no nulldb, no build ;)

Reply:
seems you have not enabled "allow changes by maintainers" or I would have just committed that change now, so that we can move on?